Commons/bungee/src/cl/bgmp/bungee/commands/staffchat/StaffChatCommands.java

```diff
@@ -0,0 +1,30 @@
+package cl.bgmp.bungee.commands.staffchat;
+
+import cl.bgmp.bungee.BungeeMessages;
+import cl.bgmp.bungee.ChatConstant;
+import com.sk89q.minecraft.util.commands.Command;
+import com.sk89q.minecraft.util.commands.CommandContext;
+import com.sk89q.minecraft.util.commands.CommandPermissions;
+import net.md_5.bungee.api.ChatColor;
+import net.md_5.bungee.api.CommandSender;
+import net.md_5.bungee.api.connection.ProxiedPlayer;
+
+public class StaffChatCommands {
+
+  @Command(
+      aliases = {"admin", "a", "staffchat", "sc"},
+      desc = "Use the admin chat.",
+      usage = "<msg>",
+      help = "Use alone to lock the chat in admin mode. More arguments will send a message only.")
+  @CommandPermissions("commons.bungee.command.admin")
+  public static void admin(final CommandContext args, CommandSender sender) {
+    if (!(sender instanceof ProxiedPlayer)) {
+      sender.sendMessage(
+          BungeeMessages.colourify(ChatColor.RED, ChatConstant.NO_CONSOLE.getAsTextComponent()));
```

Comment:
You could even add a `sender.sendConsoleWarning`(or `throw` an Exception) or something. Since you have it in so many places

Reply:
I feel this is more on sk89q's framework's end. Something to indicate whether the command is for player or console usage should be introduced.

https://github.com/ElMedievo-UdeC/Network/pull/31/commits/94183bbf59d2edb08225927f4a4f65a805a29b94 does the job anyways, I don't mind keeping this boilerplateish practice for now.